src/GameView.cpp

```diff
@@ -52,11 +52,32 @@ void GameView::update(Platform *platform, StolenObject *stolenObject, PlatformMe
 	}
 
 	if (platform){
-		//TODO: load platform sprites from resources
+		sf::RectangleShape rectangle(sf::Vector2f(platform -> width, platform -> height)); //TODO: fix
+		rectangle.setOrigin(platform -> width/2, platform -> height/2);
+		//shape.setSize(sf::Vector2f(platform->width, platform->height));
+		rectangle.setPosition(platform -> xCoord, platform -> yCoord);
+		rectangle.setFillColor(platform -> color);
+
+		//this -> drawPlatform(rectangle);
+	}
+
+	if (base){
+		sf::RectangleShape rectangle(sf::Vector2f(base -> width, base -> height)); //TODO: fix
+		rectangle.setOrigin(base -> width/2, base -> height/2);
+		//shape.setSize(sf::Vector2f(platform->width, platform->height));
+		rectangle.setPosition(base -> xCoord, base -> yCoord);
+		rectangle.setFillColor(base -> color);
+
+		//this -> drawPlatform(rectangle);
```

Comment:
committing this sort of commented code is fine, since we'll replace it, but don't commit commented code usually